File: Recursion/Fibo.py
def fibo(n):
    if n == 1 or n == 2:
        return 1
    else:
        return fibo(n - 1) + fibo(n - 2)

n = int(input("Enter a number: "))
if n < 1:
    print("Please enter a number greater than 1.")
else:
    print(f"Fibonacci of {n} is {fibo(n)}")

File: Recursion/Fibo.c
# include<stdio.h>
int Fibo(int n) {
    if (n == 1|| n == 2)
        return 1;
    else
        return Fibo(n - 1) + Fibo(n - 2);
}
int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    if (n < 1)
        printf("Please enter a positive integer.\n");
    printf("Fibonacci of %d is %d\n", n, Fibo(n));
    return 0;
}